contracts/managers/price-triggers/IPriceTrigger.sol

```diff
@@ -33,6 +33,5 @@ interface IPriceTrigger {
      */
     function getBaseAssetAllocation()
         external
```

Comment:
Should leave a comment as to why we decided to remove the view (e.g. to not allow reverts)

Reply:
Hmm not sure that's the take away. That's what caused the remove in this case but I think more generally we shouldn't be putting visibility modifiers in our Interfaces unless we are REALLY confident how they'll be used.

Reply:
All that to say I'm not sure it's worthy of a comment because it shouldn't have been done in the first place!

Reply:
Typically, getter functions (functions get begin with get) are non-state modifying. So the fact that this one has the potential to is something I would note or modify the name of.

https://stackoverflow.com/questions/13876066/is-it-bad-practice-to-have-my-getter-method-change-the-stored-value

Maybe if you used a different function name like `retrieveBaseAssetAllocation`